### PR TITLE
refactor: Make bundles only contain systems

### PR DESCRIPTION
**Pull request by Rhuagh**
This change is [Reviewable](https://reviewable.io/reviews/amethyst/amethyst/675)

### CODE REVIEW

File: examples/animation/main.rs [hunk 1]
```diff
@@ -282,7 +282,7 @@ fn initialise_sphere(world: &mut World) -> Entity {
     /*let mut nodes = HashMap::default();
```
Comment (jojolepro) [inline examples/animation/main.rs]: Why is there commented code in the repo?

**Comment by Rhuagh** [comment]
Ready.

**Comment by Rhuagh** [comment]
I can no longer build amethyst on my mac, so I will run it through CI for checking.

**Pull request merged**